### DISCUSSION TITLE
如何获取到双击节点编辑过后的值？

### DISCUSSION THREAD

**Discussion by Eohu** [hizzgdev/jsmind#626] [General]
![微信图片_20240828091309](https://github.com/user-attachments/assets/01231d12-9010-4d6f-adef-0127dc3e7d66)

**Comment by hizzgdev** [comment]
你是需要一个事件（Event）吗？可以看看这个回复： https://github.com/hizzgdev/jsmind/discussions/527#discussioncomment-7443323

**Comment by Eohu** [comment]
或者可以把这个输入框禁用掉吗，双击后不展示输入框，可以做到吗？

**Reply by hizzgdev** [reply]
可以的，加上 `enable_dblclick_handle` 这个选项就行了。

```javascript
var options = {
    container:'jsmind_container',
    editable:true,
    default_event_handle: {
        enable_dblclick_handle: false // disable mouse double click
    }
};
var jm = new jsMind(options);
```

https://github.com/hizzgdev/jsmind/discussions/595#discussioncomment-8999971 这个回复里还有一些其它的内容可供参考。

**Reply by LH1110** [reply]
请问可以支持指定的节点禁用双击事件吗

**Reply by hizzgdev** [reply]
jsmind 没有提供这个选项，但我感觉你可以尝试给指定的节点绑定双击事件，然后利用事件冒泡的特点，阻止它向上传递应该就行了。

**Reply by LH1110** [reply]
好的 我尝试一下

**Reply by LH1110** [reply]
节点边框的样式有封装变量值吗,就比如下面图片必须与非必须的颜色不一样

![Snipaste_2025-03-20_10-18-46](https://github.com/user-attachments/assets/2470d23e-7d1b-4efe-8fca-e01aba56b848)

**Reply by hizzgdev** [reply]
目前有其它样式可供使用，[这个文档](https://hizzgdev.github.io/jsmind/docs/zh/1.usage.html) 里的样式章节有介绍。
不过没有支持边框样式，你如果需要的话，可以考虑提个 PR 支持一下。

或者还有一种方法是使用 [custom_node_render](https://hizzgdev.github.io/jsmind/docs/zh/options/option.view.custom_node_render.html)，你可以了解一下看怎么使用。

**Reply by LH1110** [reply]
好的 希望考虑支持一下 ,我先看看另一种怎么使用
